frontend/profil.js

```diff
@@ -116,22 +119,6 @@ function postData(person) {
     });
 }
 
-function updateData(person) {
```

Comment:
not the good place for the function but go add a new branch with the creation of a page for editing personnal info, it could be useful as said higher